src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/notifier.html.twig

```diff
@@ -0,0 +1,168 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% set events = collector.events %}
+
+    {% if events.messages|length %}
+        {% set icon %}
+            {% include('@WebProfiler/Icon/mailer.svg') %}
+            <span class="sf-toolbar-value">{{ events.messages|length }}</span>
+        {% endset %}
+
+        {% set text %}
+            <div class="sf-toolbar-info-piece">
+                <b>Sent notifications</b>
+                <span class="sf-toolbar-status">{{ events.messages|length }}</span>
+            </div>
+
+            {% for transport in events.transports %}
+                <div class="sf-toolbar-info-piece">
+                    <b>{{ transport }}</b>
+                    <span class="sf-toolbar-status">{{ events.messages(transport)|length }}</span>
+                </div>
+            {% endfor %}
+        {% endset %}
+
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { 'link': profiler_url }) }}
+    {% endif %}
+{% endblock %}
+
+{% block head %}
+    {{ parent() }}
+    <style type="text/css">
+        /* utility classes */
+        .m-t-0 { margin-top: 0 !important; }
+        .m-t-10 { margin-top: 10px !important; }
+
+        /* basic grid */
+        .row {
+            display: flex;
+            flex-wrap: wrap;
+            margin-right: -15px;
+            margin-left: -15px;
+        }
+        .col {
+            flex-basis: 0;
+            flex-grow: 1;
+            max-width: 100%;
+            position: relative;
+            width: 100%;
+            min-height: 1px;
+            padding-right: 15px;
+            padding-left: 15px;
+        }
+        .col-4 {
+            flex: 0 0 33.333333%;
+            max-width: 33.333333%;
+        }
+
+        /* small tabs */
+        .sf-tabs-sm .tab-navigation li {
+            font-size: 14px;
+            padding: .3em .5em;
+        }
+    </style>
+{% endblock %}
+
+{% block menu %}
+    {% set events = collector.events %}
+
+    <span class="label {{ events.messages|length ? '' : 'disabled' }}">
+        <span class="icon">{{ include('@WebProfiler/Icon/mailer.svg') }}</span>
```

Comment:
- [ ] exchange icon

Reply:
@javiereguiluz Can you work on an icon for the notifier profiler?

Reply:
Sorry for being late, but our designer just sent us the icon for the notifier. It uses the "bell icon" which is universal for notifications:

```xml
<svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" viewBox="0 0 24 24"><path fill="#AAA" d="M11.23,4.8c-3,.83-5.07,3.18-5.07,5.9H7.39c0-2.21,1.77-4,4.17-4.72ZM7.49,0A12.22,12.22,0,0,0,0,11.59H2.07A10.14,10.14,0,0,1,8.23,2Zm8.24,2a10.14,10.14,0,0,1,6.16,9.64H24A12.24,12.24,0,0,0,16.47,0ZM4.41,15.64V10.7a7.57,7.57,0,0,1,15.14,0v4.94l3.3,4.4H1.11Zm4.45,5.3A3.06,3.06,0,0,0,11.92,24H12a3.07,3.07,0,0,0,3.07-3.06Z"/></svg>
```

Reply:
@javiereguiluz as this PR is already merged, it might be better if you create a new PR with the new icon instead.

Reply:
Thank you @javiereguiluz I created a new PR: #38545